MdeModulePkg/MdeModulePkg.dsc

```diff
@@ -116,11 +116,13 @@
 
 ##MSCHANGE Begin
 [LibraryClasses.X64, LibraryClasses.IA32]
+!if $(TOOLCHAIN) == VSLATESTx86

```

Comment:
this isn't totally reliable.  Might be ok for now but should really move to feature based value since this is optional feature based on compiler and compiler flags.